app/models/requirement.rb

```diff
@@ -0,0 +1,57 @@
+class Requirement < ActiveRecord::Base
+  belongs_to :task
+  belongs_to :skill
+  #has_many :assignments
+
+  validates :task, presence: true
+  validates :skill, presence: true
+
+  validates :minimum_people, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :maximum_people, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :desired_people, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validate :valid_people_numbers
+
+  STATUS_CHOICES_ARRAY = ['Empty', 'Inadequate', 'Adequate', 'Satisfied', 'Full']
```

Comment:
What is being stored in the db ? Cause if we're storing the index, changing the array would be really bad !